DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -14,479 +14,381 @@
 #define L1_TRACK_TRIGGER_TRACK_FORMAT_H
 
 #include "DataFormats/L1TrackTrigger/interface/TTStub.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTrack_TrackWord.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "DataFormats/Common/interface/DetSetVectorNew.h"
 #include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/SiStripDetId/interface/StripSubdetector.h"
```

Comment:
hi @mdhildreth  - what change requires this include?

Reply:
Hi @davidlange6. Ah, maybe this isn't required in 10X.  It definitely is in 11X, as someone reshuffled the headers.  I could strike it here if it's a concern.

Reply:
@davidlange6 In 11X the code didn't compile without that header file (I found this out by copying the 10X version an 11X release area first). I agree it's probably not needed here.